common/lib/capa/capa/tests/response_xml_factory.py

```diff
@@ -193,6 +230,12 @@ def create_response_element(self, **kwargs):
             responseparam_element = etree.SubElement(response_element, 'responseparam')
             responseparam_element.set('type', 'tolerance')
             responseparam_element.set('default', str(tolerance))
+            if partial_range is not None and 'close' in credit_type:
+                responseparam_element.set('partial_range', str(partial_range))
+
+        if partial_answers is not None and 'list' in credit_type:
+            responseparam_element = etree.SubElement(response_element, 'responseparam')
```

Comment:
@benpatterson Thanks for the info. Since that error shows up 41 times in the link you gave me for the last build, I'm going to propose that maybe this error is full of crap. I'm inclined to disable inline with a comment to explain why.